crates/iota-protocol-config/src/snapshots/iota_protocol_config__test__version_2.snap

```diff
@@ -12,6 +11,7 @@ feature_flags:
   zklogin_max_epoch_upper_bound_delta: 30
   enable_vdf: true
   passkey_auth: true
+  relocate_event_module: true
```

Comment:
should not be part of protocol 2

Reply:
Removed it from the protocol 2 at [c543c68](https://github.com/iotaledger/iota/pull/4712/commits/c543c687dadde7bb44da097b03bc0af62dc57fcc).